### PR TITLE
Pycomet master

### PR DESCRIPTION
**Pull request by Pycomet-zz**
New chad's feature and layout push

### CODE REVIEW

File: src/components/Stations.jsx [hunk 1]
```diff
@@ -8,73 +8,70 @@ const useStyles = makeStyles((theme) => ({
...
-  'EDMOND',
-  'FL',
-  'LOCAMS',
-  'NAMIBIA',
-  'NZ',
-  'SA',
-  'SONOTACO',
-  'TK',
-];
+const options = {
```
Comment (chichengren) [inline src/components/Stations.jsx:16]: Should we use the station abbreviation as the key for this hash map?

File: src/components/Footer.jsx [hunk 2]
```diff
@@ -101,15 +49,48 @@ class Footer extends Component {
...
+                this.setState({ showAbout: !this.state.showAbout })
+              }
+            >
+              About
+            </p>
+          </div>
+          <div className="footer-nav-link">Start a network</div>
+          <div className="footer-nav-link">CAMS</div>
+          <div className="footer-nav-link">
+            <React.Fragment>
```
Comment (chichengren) [inline src/components/Footer.jsx]: I think this `Fragment` is redundant. The same also applies below

File: src/components/Stations.jsx (hunk 1 above)
Comment (chichengren) [reply src/components/Stations.jsx:16]: And then we can use the abbreviation as key for all station options.

File: src/contexts/navigation.js [hunk 3]
```diff
@@ -135,6 +135,10 @@ export function NavigationProvider({ children }) {
     let finalDate = dateParam ? dateParam : date;
     let finalSource = sourceParam ? sourceParam : source;
     setDataAll(finalDate, finalSource);
+
+    retrieveMeteors(date, source);
```
Comment (chichengren) [inline src/contexts/navigation.js]: why do we need to retrieve these data again here?

Comment (Pycomet-zz) [reply src/contexts/navigation.js]: I'll take this out

File: src/components/Footer.jsx (hunk 2 above)
Comment (Pycomet-zz) [reply src/components/Footer.jsx]: Okay, taken care of too

File: src/components/Stations.jsx (hunk 1 above)
Comment (Pycomet-zz) [reply src/components/Stations.jsx:16]: I have already fixed this, using the full names as the key for the dropdown the abbreviations passed to the request

**Review by chichengren — APPROVED** [review]

**Pull request merged**